jdl/index.ts

```diff
@@ -1 +1,3 @@
 export * from './jdl-importer.js';
+export * from './parsing/api.js';
+export * from './jhipster/index.mjs';
```

Comment:
I wonder if we should export jhipster definitions separately like `jdl/jhipster`.